include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_reduce_then_scan.h

```diff
@@ -442,7 +437,7 @@ struct __parallel_reduce_then_scan_scan_submitter<
     operator()(_ExecutionPolicy&& __exec, const sycl::nd_range<1> __nd_range, _InRng&& __in_rng, _OutRng&& __out_rng,
                _TmpStorageAcc& __scratch_container, const sycl::event& __prior_event,
                const std::uint32_t __inputs_per_sub_group, const std::uint32_t __inputs_per_item,
-               const std::size_t __block_num) const
+               const std::size_t __block_num, sycl::kernel& __scan_kernel) const
```

Comment:
Can the kernel be const here or does that cause issues?

Reply:
I checked the SYCL spec and the `get_kernel_bundle` member function is const and there are no compilation issues, so I have made these references all const.